.travis.yml

```diff
@@ -29,40 +43,66 @@ matrix:
       env: TWISTED_VERSION=16.2.0
     - python: "2.7"
       env: TWISTED_VERSION=16.3.0
+    - python: "2.7"
+      env: TWISTED_VERSION=16.4.0
+    - python: "2.7"
+      env: TWISTED_VERSION=16.5.0
+    - python: "2.7"
+      env: TWISTED_VERSION=16.6.0
+    - python: "2.7"
+      env: TWISTED_VERSION=17.1.0
 
     - python: "2.7"
       env: DJANGO_VERSION=1.6.11
     - python: "2.7"
       env: DJANGO_VERSION=1.7.11
     - python: "2.7"
-      env: DJANGO_VERSION=1.8.14
+      env: DJANGO_VERSION=1.8.18
+    - python: "2.7"
+      env: DJANGO_VERSION=1.9.13
     - python: "2.7"
-      env: DJANGO_VERSION=1.9.9
+      env: DJANGO_VERSION=1.10.7
     - python: "2.7"
-      env: DJANGO_VERSION=1.10
+      env: DJANGO_VERSION=1.11.1
+
     - python: "3.3"
       env: DJANGO_VERSION=1.6.11
     - python: "3.3"
-      env: DJANGO_VERSION=1.8.14
+      env: DJANGO_VERSION=1.8.18
+
     - python: "3.4"
       env: DJANGO_VERSION=1.7.11
     - python: "3.4"
-      env: DJANGO_VERSION=1.8.14
+      env: DJANGO_VERSION=1.8.18
     - python: "3.4"
-      env: DJANGO_VERSION=1.9.9
+      env: DJANGO_VERSION=1.9.13
     - python: "3.4"
-      env: DJANGO_VERSION=1.10
+      env: DJANGO_VERSION=1.10.7
+    - python: "3.4"
+      env: DJANGO_VERSION=1.11.1
+
     - python: "3.5"
-      env: DJANGO_VERSION=1.8.14
+      env: DJANGO_VERSION=1.8.18
     - python: "3.5"
-      env: DJANGO_VERSION=1.9.9
+      env: DJANGO_VERSION=1.9.13
     - python: "3.5"
-      env: DJANGO_VERSION=1.10
+      env: DJANGO_VERSION=1.10.7
+    - python: "3.5"
+      env: DJANGO_VERSION=1.11.1
+
+    - python: "3.6"
+      env: DJANGO_VERSION=1.8.18
+    - python: "3.6"
+      env: DJANGO_VERSION=1.9.13
+    - python: "3.6"
+      env: DJANGO_VERSION=1.10.7
+    - python: "3.6"
+      env: DJANGO_VERSION=1.11.1
 
 install:
   - if [ -v FLASK_VERSION ]; then pip install Flask==$FLASK_VERSION; fi
   - if [ -v TWISTED_VERSION ]; then pip install Twisted==$TWISTED_VERSION treq; fi
   - if [ -v DJANGO_VERSION ]; then pip install Django==$DJANGO_VERSION; fi
 
 script:
-  - python setup.py test
+  - python -Wall setup.py test
```

Comment:
Eh, let's strip this out for now. I'll look into this a bit later though I would wager that the tests would pass just fine with it there.